packages/amplify-graphql-docs-generator/src/index.ts

```diff
@@ -24,22 +24,17 @@ const FILE_EXTENSION_MAP = {
 }
 
 function generate(
-  schemaPath: string,
-  outputPath: string,
-  options: { separateFiles: boolean; language: string; maxDepth: number }
-): void {
+  schema: IntrospectionQuery,
```

Comment:
The codegen plugin in expects the default export to accept the `schemaPath` and `outputPath`. Could you export a `cliFileHandler` and use that in the codegen 
https://github.com/aws-amplify/amplify-cli/blob/a61237ff51a26fbf93ee423b43a34d89c06acf57/packages/amplify-codegen/src/commands/statements.js#L3-L5

https://github.com/aws-amplify/amplify-cli/blob/a61237ff51a26fbf93ee423b43a34d89c06acf57/packages/amplify-codegen/src/commands/statements.js#L37-L45

Reply:
👍 Good spot, fixed in a87e413